server/spec/auth/authenticate_user_spec.rb

```diff
@@ -0,0 +1,32 @@
+# spec/auth/authenticate_user_spec.rb
+require 'rails_helper'
```

Comment:
You could add this to the `spec_helper` to avoid needing to do this in every _spec.

Reply:
Should this be under the Rspec config section as config.include RailsHelper?

Reply:
No need to respond to this